winiety_web_app/src/routes/Home/Home.tsx

```diff
@@ -0,0 +1,53 @@
+import { Container, CssBaseline, Button } from '@material-ui/core';
+import React, { ReactElement } from 'react';
+import { useStoreActions, useStoreState } from 'store';
+import useStyles from './styles';
+
+const Home = (): ReactElement => {
+  const classes = useStyles();
+  const login = useStoreActions((actions) => actions.userSession.initiateLogin);
+  const logout = useStoreActions(
+    (actions) => actions.userSession.initiateLogout
+  );
+  const isAuthenticated = useStoreState(
+    (state) => state.userSession.isAuthenticated
+  );
+
+  return (
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
+      <div className={classes.paper}>
+        <h1>Winiety</h1>
+        {isAuthenticated ? (
```

Comment:
Generalnie jak ktoś jest zalogowany, to bym nic nie pokazywał, tylko przekierował na jakąś stronkę